dev/PipsPager/PipsPager.cpp

```diff
@@ -516,6 +477,93 @@ void PipsPager::OnNextButtonClicked(const IInspectable& sender, const winrt::Rou
     SelectedPageIndex(SelectedPageIndex() + 1);
 }
 
+void PipsPager::OnGotFocus(const winrt::RoutedEventArgs& args)
+{
+    if (const auto btn = args.OriginalSource().try_as<winrt::Button>())
+    {
+        // If the element inside the Pager is already keyboard focused
+        // and the user will use the mouse to focus on something else
+        // the LostFocus will not be triggered on keyboard focused element
+        // while GotFocus will be triggered on the new mouse focused element.
+        // We account for this scenario and update m_isFocused in case
+        // user will use mouse while being in keyboard focus.
```

Comment:
Because if the focus is already inside the Pager (keyboard focus) and the user will click on any element inside with the mouse - GotFocus will be raised again without raising LostFocus : )

Reply:
But then focus would still be inside so it should be fine right? There wouldn't be a change in the "focus inside PipsPager" department if I understand this correctly.

Reply:
Imagine this case: user tabbed into the pager and we see the focus rect. Then, while still having focus (with focust rect on) the user clicks on any element inside the Pager with the mouse - the GotFocus will be raised and the focus rect will disappear. Then, if the user moves the mouse away - what should we do: leave the nav buttons visible (because technically the keyboard focus is still there) or make them disappear? I've talked to @gabbybilka and we decided that it makes more sense to make buttons invisible again because the user doesn't see the focus rect and the mouse is not over the pager - so technically there's no "visible" focus anymore : )
Do you think otherwise?

Reply:
We can probably rename the `m_isFocused` to `m_isKeyboardFocused` for it to make more sense?:)

Reply:
Hmm yeah I think that's a good way to handle this, Yes I think renaming m_isFocused is a good idea since we actually care about keyboard focus, not focus in general.